apps/admin_api/test/admin_api/v1/controllers/admin_auth/transaction_request_controller_test.exs

```diff
@@ -299,16 +307,8 @@ defmodule AdminAPI.V1.AdminAuth.TransactionRequestControllerTest do
           address: "FAKE-0000-0000-0000"
         })
 
-      assert response == %{
-               "success" => false,
-               "version" => "1",
-               "data" => %{
-                 "code" => "wallet:wallet_not_found",
-                 "description" => "There is no wallet corresponding to the provided address",
-                 "messages" => nil,
-                 "object" => "error"
-               }
-             }
+      assert response["success"] == false
```

Comment:
Yes! Let's start asserting specific fields in the controller, rather than the full response map (since they're already covered by serializer tests and we are starting e2e tests now)